test/fixtures/relationship_error.json

```diff
@@ -0,0 +1,17 @@
+[
```

Comment:
I would like to have 3 samples:

1. missing `source_ref`, present `rel_type`  and `target_ref` (ok, sample 1)
2. missing `rel_type`, present `source_ref` and `target_ref` (no, likely sample 2)
3. missing `target_ref`, present other 2 (missing)